torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -811,17 +814,24 @@ def benchmark_one_config(config):
         best_config = self.coordesc_tuner.autotune(
             benchmark_one_config, launcher.config, None
         )
-        time_taken_ns = time.time_ns() - start_time
+        coordesc_time_taken_ns = time.time_ns() - start_time
         best_config.found_by_coordesc = True
 
         if self.save_cache_hook:
-            self.save_cache_hook(best_config, time_taken_ns, found_by_coordesc=True)
+            self.save_cache_hook(
+                best_config,
+                getattr(self, "autotune_time_taken_ns", 0) + coordesc_time_taken_ns,
```

Comment:
why not set the default of this attr to 0 so that you dont need to do this getattr business?